Configurations/50-nonstop.conf

```diff
@@ -0,0 +1,176 @@
+#### Nonstop configurations
+    "nonstop-common" => {
+        inherit_from     => [ "BASE_unix" ],
+        template         => 1,
+        cc               => "c99",
+        cflags           => add_before(picker(debug   => "-g -O0",
+                                              release => "-g -O2") ,"-Wextensions -Wnowarn=203,220,272,734,770,1506 -Wbuild_neutral_library"),
+        perl             => "/usr/bin/perl",
+        lflags           => "-lrld",
+        shared_target    => "self",
+        shared_cflag     => "",
+        shared_ldflag    => "-Wshared",
+        shared_extension => ".so",
+        enable           => ["egd"],
+        dso_scheme       => "DLFCN",
+    },
+    "nonstop-nsx" => {
+        inherit_from     => [ "nonstop-common" ],
+        cflags           => sub { join(" ",@_,"-Wverbose -I/usr/local/include") },
+        shared_cflag     => sub { join(" ",@_,"-Wverbose -I/usr/local/include") },
```

Comment:
`shared_cflag` are additional flags for shared libraries and modules.  You shouldn't need to copy the `cflags` value here.